browser/speedreader/speedreader_tab_helper.h

```diff
@@ -88,9 +88,9 @@ class SpeedreaderTabHelper
   // mojom::SpeedreaderHost:
   void OnShowOriginalPage() override;
 
-  void ChangeTheme(const std::string& theme);
-  std::string GetSelectedTheme();
-  std::string GetSystemTheme();
+  void SetTheme(Theme theme);
+  Theme GetTheme();
+  std::string GetThemeName();
```

Comment:
why do we need `::GetThemeName` here again? We're probably using the service method or is this a left over?

Reply:
Yep, should be deleted.